packages/core/tests/span.test.ts

```diff
@@ -196,6 +196,123 @@ describe('Span', () => {
       expect(spanContextEquals(spanIsNotContext, client.currentSpanContext)).toBe(false)
       expect(spanContextEquals(spanIsContext, client.currentSpanContext)).toBe(true)
     })
+
+    it('sets traceId and parentSpanId from parentContext if specified', async () => {
+      let count = 0
+      const idGenerator = {
+        generate (bits: 64 | 128) {
+          if (bits === 64) {
+            count++
```

Comment:
If `count` is only incremented here, doesn't this somewhat invalidate the test since all spans would always be in the same trace?

Reply:
doh, yes it does- good spot. That's the whole point of having that generator! Updated so that a new span ID and trace ID is generated each time